app/routes/content.js

```diff
@@ -104,7 +104,8 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
                 ['og:description', node.get('description')],
                 ['og:site_name', this.get('theme.provider.name')],
                 ['og:type', 'article'],
-                ['article:published_time', new Date(preprint.get('dateCreated') || null).toISOString()]
+                ['article:published_time', new Date(preprint.get('datePublished')).toISOString()],
```

Comment:
Can you clarify why this extra conversion step is necessary? (lines 107-108)

Within the preprints front-end, is there ever a situation where `datePublished` and/or `dateModified` would be null? If not, then any DS.attr of type `date` will [already be a date object](https://github.com/emberjs/data/blob/master/addon/-private/transforms/date.js#L27). (1)


If I'm reading this right, that means:
- The only reason to convert the field to a `Date` would be if Date was null- eg if you weren't guaranteed that the field value had a `toUTCString` method already.
- If you create a Date by passing in null, then the date displayed might not be optimal for the use case. (2)


[1] Example command line exploration by inserting a debugger statement into content.js...
```
datepub = preprint.get('datePublished')
Wed Jan 18 2017 18:17:56 GMT-0500 (EST)
datepub instanceof Date
true
```

[2] Sample date creation from null
```
new Date(null).toISOString()
"1970-01-01T00:00:00.000Z"
```